ksql-engine/src/main/java/io/confluent/ksql/services/KafkaClusterUtil.java

```diff
@@ -35,6 +38,18 @@ private KafkaClusterUtil() {
 
   }
 
+  public static boolean isAuthorizedOperationsSupported(final AdminClient adminClient) {
+    try {
+      final DescribeClusterResult authorizedOperations = adminClient.describeCluster(
+          new DescribeClusterOptions().includeAuthorizedOperations(true)
+      );
+
+      return authorizedOperations.authorizedOperations().get() != null;
+    } catch (Exception e) {
```

Comment:
do we need to check for `ClusterAuthorizationException` like we do above?

Reply:
I don't think so. By the time this method is called, the `TopicAccessValidatorFactory` already checked if the user is authorized to access the cluster.